apple/swift-docc-render
# Add Skeleton screen to Navigator

Bug/issue #90172650, if applicable: 

## Summary

Add Skeleton screen to Navigator

https://user-images.githubusercontent.com/8567677/193631610-c4a0bffa-3110-4759-a7d2-7e8154548e5e.mov

## Dependencies

NA

## Testing

Steps:
1. Open the project and go to `src/components/Navigator.vue`
2. Set line 17 to `:isLoading="true"`
3. Run the project
4. Assert that you can see the skeleton screen

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
